cdpetron.py

```diff
@@ -348,7 +352,7 @@ def main(branch_dir, dump_dir, language, lang_config, imag_config,
         try:
             lang_config = _config[args.language]
         except KeyError:
-            print("ERROR: there's no %r in language config file %r" % (
+            logger.error("there's no %r in language config file %r" % (
                   args.language, _config_fname))
```

Comment:
No armes el string vos si estás usando logging, aprovechá que logging mismo lo arma de forma más robusta. 

O sea, hacé 

    logger.error("there's no %r in language config file %r", args.language, _config_fname)

Reply:
Joya. En realidad lo único que hice fue cambiar los prints por logging, por eso no lo noté. Recién vi que el resto estaban bien originalmente.